go/protocol/chat1/gregor.go

```diff
@@ -18,24 +18,32 @@ type NewConversationPayload struct {
 }
 
 type NewMessagePayload struct {
-	Action    string         `codec:"Action" json:"Action"`
-	ConvID    ConversationID `codec:"convID" json:"convID"`
-	Message   MessageBoxed   `codec:"message" json:"message"`
-	InboxVers InboxVers      `codec:"inboxVers" json:"inboxVers"`
+	Action        string         `codec:"Action" json:"Action"`
+	ConvID        ConversationID `codec:"convID" json:"convID"`
+	Message       MessageBoxed   `codec:"message" json:"message"`
+	InboxVers     InboxVers      `codec:"inboxVers" json:"inboxVers"`
+	UnreadUpdates []UnreadUpdate `codec:"unreadUpdates" json:"unreadUpdates"`
```

Comment:
You're right it'll almost always contain 1 element. I was thinking that if there's an error computing the update, it'll just be empty. And if in the future we batch them or do something fancy, it could be easier if it's an array. Could change it to be single, up to you.